src/common/consts/tokens.ts

```diff
@@ -45,3 +45,14 @@ export const WRAPPED_NATIVE_CURRENCIES: Record<SupportedChainId, TokenInfo> = {
     address: '0xfFf9976782d46CC05630D1f6eBAb18b2324d6B14',
   },
 }
+
+/**
+ * Just a base template for the native currency, handy to define new networks.
+ */
+export const nativeCurrencyTemplate: Omit<TokenInfo, 'chainId'> = {
+  address: NATIVE_CURRENCY_ADDRESS,
+  decimals: 18,
+  name: 'Ether',
+  symbol: 'ETH',
+  logoUrl: WRAPPED_NATIVE_CURRENCIES[SupportedChainId.MAINNET].logoUrl,
```

Comment:
As I mentioned for Gnosis chain.
The wrapped token logo is different from native token logo, no?

Reply:
Yes, good catch: 97fdada316f4e70db1eb64ad776a3eb9f3324c15